app/tasks/dynamic_vector_tile_cache_assets.py

```diff
@@ -7,6 +7,7 @@
 from ..models.enum.assets import AssetStatus, is_database_asset
 from ..models.enum.change_log import ChangeLogStatus
 from ..models.enum.creation_options import IndexType
+from ..models.orm.asset_metadata import FieldMetadata
```

Comment:
In the other place where this is imported, it is done as:

```
from ..models.orm.asset_metadata import FieldMetadata as ORMFieldMetadata
```

so you might want to change to that (and change to ORMFieldMetadata below) for consistency.

Reply:
Thanks Dan! Although I think it has generally been done as a necessity when things conflict, you're right that there's value in consistency. I'll change it!